src/SourceBuild/content/build.sh

```diff
@@ -261,3 +271,9 @@ else
 
   "$CLI_ROOT/dotnet" msbuild "$SCRIPT_ROOT/build.proj" -bl:"$SCRIPT_ROOT/artifacts/log/Debug/Build_$LogDateStamp.binlog" -flp:"LogFile=$SCRIPT_ROOT/artifacts/logs/Build_$LogDateStamp.log" ${MSBUILD_ARGUMENTS[@]} "$@"
 fi
+
+# Clean up the temporary SDK if it was used
+if [ -d "$CustomDotNetSdkDir" ]; then
```

Comment:
Two UX issues come to mind with using a tmp directory that is automatically cleaned up.

1. It can hurt diagnostics specifically related to the SDK resolver.  
2. When doing repetitive builds during development it is unfortunate to always be coping/cleaning the SDK.

Would it make sense to copy it into the .dotnet folder that would be setup if you let prep initialize the sdk and never clean-it up?

Reply:
I think that makes more sense.